src/assets/Assets.ts

```diff
@@ -71,22 +71,30 @@ export interface AssetInitOptions
  * Only one Asset Class exists accessed via the Global Assets object. See [here]{@link assets.Assets},
  * for full documentation.
  *
- * It has four main responsibilities:
- * <br>     1. Allows users to map URLs to keys and resolve them according to the user's browser capabilities
- * <br>     2. Loads the resources and transforms them into assets that developers understand.
- * <br>     3. Caches the assets and provides a way to access them.
- * <br>     4. Allow developers to unload assets and clear the cache.
- *
- * It also has a few advanced features:
- * <br>1. Allows developers to provide a manifest upfront of all assets and help manage them via 'bundles'.
- * <br>2. Allows users to background load assets. Shortening (or eliminating) load times and improving UX. With this feature,
+ * <br>
+ *
+ * **It has four main responsibilities:**
+ * 1. Allows users to map URLs to keys and resolve them according to the user's browser capabilities
+ * 2. Loads the resources and transforms them into assets that developers understand.
+ * 3. Caches the assets and provides a way to access them.
+ * 4. Allow developers to unload assets and clear the cache.
+ *
+ * <br>
+ *
+ * **It also has a few advanced features:**
+ * 1. Allows developers to provide a manifest upfront of all assets and help manage them via 'bundles'.
+ * 2. Allows users to background load assets. Shortening (or eliminating) load times and improving UX. With this feature,
  * in-game loading bars can be a thing of the past!
  *
- * ### Assets Loading
+ * <br>
+ *
+ * <details id="assets-loading">
+ *
+ * <summary>Assets Loading</summary>
  *
- * Do not be afraid to load things multiple times - under the hood, it will NEVER load anything more than once.
+ * Do not be afraid to load things multiple times - under the hood, it will **NEVER** load anything more than once.
  *
- * For example:
+ * <u>For example:</u>
```

Comment:
This is not semantically correct usage of UL. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

Do you mean to use a header here instead?

Reply:
<u> here was intended for an underline but it seems that it is actually a misuse according to [source](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/u). I have switched to use italic instead for this 🙏

Reply:
Oops, I read that without my glasses on! Sorry, but I agree using italic is better here.